styles.css

```diff
@@ -56,7 +56,16 @@ nav {
   height: 32px;
   margin-top: 0.28rem;
 }
-
+.reset {
```

Comment:
the whole #filter-btns container seems to be too close to main heading. Try to add a bit more spacing between `Latest topics` header and this container. Not much though, couple of pixels should be enough.